stdeb.cfg

```diff
@@ -1,6 +1,6 @@
 [DEFAULT]
 No-Python2:
-Depends3: python3-setuptools, python3-yaml
+Depends3: python3-yaml
```

Comment:
In other ros-infrastructure packages, we use conditionals like `python3 (>= 3.8) | python3-importlib-metadata` to maintain deb support for older platforms, but that doesn't really work here because of the explicit version coupling to use setuptools on older platforms. None of the suites listed here have Python older than 3.9, so I just moved the minimum declared version up instead.